pkg/bot/interactive/message.go

```diff
@@ -2,13 +2,14 @@ package interactive
 
 import (
 	"fmt"
+	"github.com/slack-go/slack"
 )
 
 // Message represents a generic message with interactive buttons.
-// NOTE: for now it's unknown how this will be consumed by other communication platforms and API may change in the near future.
 type Message struct {
 	Base
-	Sections []Section
+	Sections          []Section
+	OnlyVisibleForYou string
```

Comment:
Currently, it's a dead code (the value of it), so please change it for a `bool` 👍